### PR TITLE
You can now photocopy cat ass

### PR DESCRIPTION
**Pull request by Bibby0110**
# General Documentation

### Intent of your Pull Request

Equality.

_Stolen from TG_

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Equality

# Changelog

:cl:  
rscadd: You may now photocopy cat ass
/:cl:

### CODE REVIEW

File: code/modules/paperwork/photocopier.dm [hunk 1]
```diff
@@ -133,6 +133,8 @@
 						temp_img = icon('icons/ass/assalien.png')
 					else if(ishuman(ass)) //Suit checks are in check_ass
 						temp_img = icon(ass.gender == FEMALE ? 'icons/ass/assfemale.png' : 'icons/ass/assmale.png')
```
Comment (Bibby0110) [inline code/modules/paperwork/photocopier.dm:135]:
```suggestion
						temp_img = icon(ass.gender == FEMALE ? 'icons/ass/assfemale.png' : 'icons/ass/assmale.png')
						var/mob/living/carbon/human/H = ass
```

File: code/modules/paperwork/photocopier.dm [hunk 2]
```diff
@@ -133,6 +133,8 @@
...
+						if(iscatperson(ass))
```
Comment (Bibby0110) [inline code/modules/paperwork/photocopier.dm:136]:
```suggestion
						if(iscatperson(H))
```

File: code/modules/paperwork/photocopier.dm [hunk 3]
```diff
@@ -133,6 +133,9 @@
 						temp_img = icon('icons/ass/assalien.png')
 					else if(ishuman(ass)) //Suit checks are in check_ass
 						temp_img = icon(ass.gender == FEMALE ? 'icons/ass/assfemale.png' : 'icons/ass/assmale.png')
+						var/mob/living/carbon/human/H = ass
```
Comment (alexkar598) [inline code/modules/paperwork/photocopier.dm]: Remove this and rebase onto master, I merged #11669

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/photocopier.dm (hunk 3 above)
Comment (Bibby0110) [reply code/modules/paperwork/photocopier.dm]: Never

**Review by alexkar598 — APPROVED** [review]

**Comment by Mqiib** [comment]
Add the rest of the asses you coward

**Comment by JamieD1** [comment]
Sigh
